cli/src/multisig.rs

```diff
@@ -18,11 +18,10 @@ use solana_sdk::signature::{Keypair, Signature, Signer};
 use solana_sdk::system_instruction;
 use solana_sdk::sysvar;
 
-use lido::instruction::AddMaintainerMeta;
-use lido::instruction::AddValidatorMeta;
-use lido::instruction::ChangeRewardDistributionMeta;
-use lido::instruction::LidoInstruction;
-use lido::instruction::RemoveMaintainerMeta;
+use lido::instruction::{
+    AddMaintainerMeta, AddValidatorMeta, ChangeRewardDistributionMeta, DeactivateValidatorMeta,
+    LidoInstruction, RemoveMaintainerMeta,
+};
 use lido::state::FeeRecipients;
 use lido::state::Lido;
 use lido::state::RewardDistribution;
```

Comment:
Can you also merge the rest of these modules?

Reply:
Done.